### DISCUSSION TITLE
Request to add 75 percentile under Model latency section as shown in image below

### DISCUSSION THREAD

**Discussion by ssubburaj85** [langfuse/langfuse#1190] [Ideas]
### Describe the feature or potential improvement

![image](https://github.com/langfuse/langfuse/assets/84601301/5b75e004-2142-40dc-8044-133a5c2f4147)

**Comment by marcklingen** [comment]
Just added this, will be in the next release: https://github.com/langfuse/langfuse/pull/1195

**Reply by ssubburaj85** [reply]
Thank you
